packages/store/tools/arb-passable.js

```diff
@@ -0,0 +1,89 @@
+// @ts-check
+import { Far, makeTagged } from '@endo/marshal';
+import { fc } from '@fast-check/ava';
+import '../src/types.js';
+
+/**
+ * The only elements with identity. Everything else should be equal
+ * by contents.
+ */
+export const exampleAlice = Far('alice', {});
+export const exampleBob = Far('bob', {});
+export const exampleCarol = Far('carol', {});
+
+export const arbString = fc.oneof(fc.string(), fc.fullUnicodeString());
+
+export const arbLeaf = fc.oneof(
+  fc.record({}),
+  arbString,
+  arbString.map(s => Symbol.for(s)),
+  // primordial symbols and registered lookalikes
+  fc.constantFrom(
+    ...Object.getOwnPropertyNames(Symbol).flatMap(k => {
+      const v = Symbol[k];
+      if (typeof v !== 'symbol') return [];
+      return [v, Symbol.for(k), Symbol.for(`@@${k}`)];
+    }),
+  ),
+  fc.bigInt(),
+  fc.integer(),
+  fc.constantFrom(-0, NaN, Infinity, -Infinity),
+  fc.constantFrom(null, undefined, false, true),
+  fc.constantFrom(exampleAlice, exampleBob, exampleCarol),
+  arbString.map(s => new Error(s)),
+  // unresolved promise
+  fc.constant(new Promise(() => {})),
+);
+
+const { arbDag } = fc.letrec(tie => {
+  return {
+    arbDag: fc.oneof(
+      {
+        // depthFactor: 0.5, // TODO Replace with what from new API?
+        withCrossShrink: true,
+      },
```

Comment:
There are guides for `depthFactor` → `depthSize` at [Arbitraries](https://github.com/dubzzz/fast-check/blob/a6fd30cedbbcf32d2376a5eff2cfe2c4c7a6dc50/packages/fast-check/documentation/Arbitraries.md#depth-size-explained) and [Migration 2.x to 3.x](https://github.com/dubzzz/fast-check/blob/c2600091f0410b301142254cde5cc61cd0385f78/packages/fast-check/MIGRATION_2.X_TO_3.X.md#depthsize-the-new-name-for-depthfactor-%EF%B8%8F), but [`oneof` documentation](https://github.com/dubzzz/fast-check/blob/a6fd30cedbbcf32d2376a5eff2cfe2c4c7a6dc50/packages/fast-check/documentation/Arbitraries.md#:~:text=oneof) suggests that it is necessary anyway: "_probability to select the first arbitrary will increase as we go deeper in the tree_" (which I also confirmed manually).
```suggestion
      { withCrossShrink: true },
```

Reply:
I didn't really understand the explanation, but I took your advice. Done.